src/modules/storage.js

```diff
@@ -0,0 +1,37 @@
+const addToLocalStorage = (task, completed, index) => {
+  const tasks = { task, completed, index };
+
+  const tasksList = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+
```

Comment:
- [ OPTIONAL] I noticed that on page reload, the tasks saved in the localStorage are not gotten and rendered to the `DOM`. This is not good for the user experience because a user will think that the data inputted is lost whereas the data is saved in the `localStorage` but not fetched on page reload. It would be nice to fix this issue as it will make your application more user-friendly. 👍